.eslintrc.js

```diff
@@ -27,6 +27,14 @@
         "airbnb",
     ],
     "rules": {
+        "no-new": [0],
```

Comment:
Why do we need to disable rules? Why don't fix the code?

Reply:
@nmanovic 
We can't fix the code. If we assign an expression 'new Class()' to any variable which is unusable, we got eslint error "unusable variable" . If we don't assign this expression to a variable, we got eslint error "new is not needed". If we remove new, we got syntax error and code doesn't work.